diff --git a/android/build.gradle b/android/build.gradle
index af810e8e9287a142b7b951745e1a9641acf8df0b..e2fd1ee3a362d445d517c0824393b215fea5454e 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -18,7 +18,7 @@ def isNewArchitectureEnabled() {
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'org.jetbrains.kotlin.android'
+// apply plugin: 'org.jetbrains.kotlin.android'
 if (isNewArchitectureEnabled()) {
   apply plugin: "com.facebook.react"
 }
@@ -54,7 +54,7 @@ android {
             }
         }
     }
-    kotlinOptions {
+    // kotlinOptions {
         jvmTarget = '17'
     }
 }
