diff --git a/android/build.gradle b/android/build.gradle
index af810e8e9287a142b7b951745e1a9641acf8df0b..f7f430deb8b598058fe98239ea18576a6c77f9dd 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -18,7 +18,7 @@ def isNewArchitectureEnabled() {
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'org.jetbrains.kotlin.android'
+// apply plugin: 'org.jetbrains.kotlin.android'
 if (isNewArchitectureEnabled()) {
   apply plugin: "com.facebook.react"
 }
@@ -54,9 +54,9 @@ android {
             }
         }
     }
-    kotlinOptions {
-        jvmTarget = '17'
-    }
+    // kotlinOptions {
+    //     jvmTarget = '17'
+    // }
 }
 
 repositories {
