diff --git a/android/build.gradle b/android/build.gradle
index 6348d17da8169193477781fd65a4b4532d3f8798..d6944a49d01406ed0ba9ea7371d62b0f0b78d64c 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -64,6 +64,7 @@ if (isNewArchitectureEnabled()) {
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
+
 if (project == rootProject) {
     apply from: "spotless.gradle"
 }
@@ -90,7 +91,7 @@ def shouldUseCommonInterfaceFromReanimated() {
 
     def (major, minor, patch) = getExternalLibVersion(reanimated)
     
-    return (major == 2 && minor >= 3) || major >= 3
+    return (major == 2 && minor >= 3) || major == 3
 }
 
 def shouldUseCommonInterfaceFromRNSVG() {
