diff --git a/android/build.gradle b/android/build.gradle
index 6348d17da8169193477781fd65a4b4532d3f8798..ab8f47749628ad7c808e1c6c63732c481a0e5b36 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -64,6 +64,7 @@ if (isNewArchitectureEnabled()) {
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
+
 if (project == rootProject) {
     apply from: "spotless.gradle"
 }
diff --git a/android/reanimated/src/main/java/com/swmansion/gesturehandler/ReanimatedEventDispatcher.kt b/android/reanimated/src/main/java/com/swmansion/gesturehandler/ReanimatedEventDispatcher.kt
index a7bf6aa39e1412a68c9465ab0f05f9f912b8bec0..60eef108dc279867f8a3f3df3794bcf5c35619b8 100644
--- a/android/reanimated/src/main/java/com/swmansion/gesturehandler/ReanimatedEventDispatcher.kt
+++ b/android/reanimated/src/main/java/com/swmansion/gesturehandler/ReanimatedEventDispatcher.kt
@@ -2,16 +2,9 @@ package com.swmansion.gesturehandler
 
 import com.facebook.react.bridge.ReactContext
 import com.facebook.react.uimanager.events.Event
-import com.swmansion.reanimated.ReanimatedModule
-
 class ReanimatedEventDispatcher {
-  private var reanimatedModule: ReanimatedModule? = null
-
+  @Suppress("UNUSED_PARAMETER")
   fun <T : Event<T>> sendEvent(event: T, reactApplicationContext: ReactContext) {
-    if (reanimatedModule == null) {
-      reanimatedModule = reactApplicationContext.getNativeModule(ReanimatedModule::class.java)
-    }
-
-    reanimatedModule?.nodesManager?.onEventDispatch(event)
+    // no-op on new architecture and Reanimated 4
   }
 }
